src/PostgreSQLCopyHelper/Model/TargetColumn.cs

```diff
@@ -11,6 +11,8 @@ public class TargetColumn
 
         public Type ClrType { get; internal set; } 
 
-        public NpgsqlDbType DbType { get; internal set; }
+        public NpgsqlDbType? DbType { get; internal set; }
+
+        public string DataTypeName { get; internal set; }
```

Comment:
I think while this is technically a breaking change, this is designed to be more of an "audit type" than a "functional type" I'm fine leaving this as a 2.8 instead of a 3.0 but I don't have too much of a strong opinion there

Reply:
Yes, well I am afraid, if you are using the ``default(NpgsqlDbType)`` it defaults to ``AbsTime`` or something like it. This would be misleading, because it isn't set after all. Let's leave it at 2.8. I think it would be best to switch to 3.0, when we change to .NET 5 or something like.